mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -314,109 +328,73 @@ export const Columns = (props: ColumnsProps) => {
         );
       },
     }),
-    columnHelper.accessor("studentComments", {
-      header: "Student comments",
+    columnHelper.accessor("comments", {
+      header: "Comments",
 
       /**
-       * Cell with StudentComments items
+       * Cell with Comments items
        */
       cell: ({row}) => {
+        const userUuid = sessionStorage.getItem("user") ?? "";
 
         /**
-         * Create StudentComment
+         * Create Comment
          */
-        const createStudentComment = async () => {
-          await DayReportDAL.createStudentComment(row.original);
-          const studentComments = [...row.original.studentComments, UnicodeSymbols.ZERO_WIDTH_SPACE];
-          const updatedDayReport = {...row.original, studentComments};
+        const createComment = async (uuid: string, way: WayPreview) => {
+          const comment = await CommentDAL.createComment(row.original, uuid, way);
```

Comment:
createComment should not change way, if it is a workaround leave a comment

Reply:
Deleted way in props